help/change-the-channel-description.md

```diff
@@ -46,6 +46,13 @@ disabled. Use Markdown formatting to include a link to a website, Zulip
 
 {!automated-notice-channel-event.md!}
 
+## Related articles
+
+* [User roles](/help/user-roles)
+* [Channel permissions](/help/channel-permissions)
+* [Rename a channel](/help/rename-a-channel)
```

Comment:
I'd order it as:

* [Rename a channel](/help/rename-a-channel)
* [Markdown formatting][markdown-formatting]
* [Channel permissions](/help/channel-permissions)